Production/govuk_ios/Views/AnimationView.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+import UIKit
+
+import Lottie
+
+class AnimationView: UIView {
+    @Inject(\.lottieConfiguration) private var config: LottieConfiguration
+
+    private lazy var internalAnimationView: LottieAnimationView = LottieAnimationView(
+        name: resourceName,
+        configuration: config
+    )
+
+    private let resourceName: String
+
+    init(resourceName: String) {
+        self.resourceName = resourceName
+        super.init(frame: .zero)
+        configureUI()
+        configureConstraints()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func configureUI() {
+        internalAnimationView.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(internalAnimationView)
+    }
+
+    private func configureConstraints() {
+        internalAnimationView.topAnchor.constraint(equalTo: topAnchor).isActive = true
```

Comment:
minor, but perhaps would be best to wrap these in a  NSLayoutConstraint.activate([]), may save you abit of code without having to write isActive = true